yo/static_content.py

```diff
@@ -0,0 +1,61 @@
+from aiohttp import web
```

Comment:
Traditionally this is how I have handled it:

1 - A prefix /static for all static content
2 - The python code can serve up static content (for easier dev testing)
3 - Nginx is configured to handle all /static requests, only passing others to the python code (which uses FastCGI in my previous projects)

In Yo in particular, some of the "static" content actually needs modifying at startup, so I made it take place entirely in the python code.

The standard template i'm thinking of would handle it like this:

1 - Prefixes for different HTML pages configured in a config file
2 - Template variable substitution for static pages done as part of build process
3 - Nginx alone handles static pages

Then standard prefixes for different content types like /static/js /static/css /static/json /static/img etc which are pure nginx.

Dynamic pages (those which might change per request) can use the same templating engine, but obviously need regenerating inside of the python code.